rerun_cpp/tests/generated/components/affix_fuzzer10.hpp

```diff
@@ -28,11 +27,11 @@ namespace rerun {
           public:
             AffixFuzzer10() = default;
 
-            AffixFuzzer10(std::optional<std::string> _single_string_optional)
-                : single_string_optional(std::move(_single_string_optional)) {}
+            AffixFuzzer10(std::optional<std::string> single_string_optional_)
+                : single_string_optional(single_string_optional_) {}
```

Comment:
This should be `single_string_optional(std::move(single_string_optional_))`, no?

Reply:
yes! Oof looks like I inverted the move/not-move logic last minute 😬

Reply:
fixed! Man, the _one_ thing that matters 🤦